nx584/client.py

```diff
@@ -23,20 +23,22 @@ def list_partitions(self):
         except TypeError:
             return r.json()['partitions']
 
-    def arm(self, armtype='auto'):
+    def arm(self, armtype='auto', partition = 1):
         if armtype not in ['stay', 'exit', 'auto']:
             raise Exception('Invalid arm type')
         r = self._session.get(
             self._url + '/command',
             params={'cmd': 'arm',
-                    'type': armtype})
+                    'type': armtype,
+                    'partition': int(partition)})
```

Comment:
This is the wrong place to do the type conversion, so please remove the `int()` here